src/lib/optimizer/strategy/expression_reduction_rule.cpp

```diff
@@ -232,7 +232,9 @@ void ExpressionReductionRule::remove_duplicate_aggregate(
     std::vector<std::shared_ptr<AbstractExpression>>& input_expressions,
     const std::shared_ptr<AbstractLQPNode>& aggregate_node, const std::shared_ptr<AbstractLQPNode>& root_node) {
   // Create a list of all sums, counts, and averages in the aggregate node.
-  std::vector<std::reference_wrapper<const std::shared_ptr<AbstractExpression>>> sums, counts, avgs;
+  std::vector<std::reference_wrapper<const std::shared_ptr<AbstractExpression>>> sums;
```

Comment:
Looks somewhat ridiculous here.

Reply:
In this place, yes. I still agree with the general rule. Is it worth making an exception here?

Reply:
Just looks stupid. But all the other places make sense.

So let's keep it this way.